app/query_param.go

```diff
@@ -21,7 +23,13 @@ func ParseHistoryQueryParams(c echo.Context, defaultLimit int64) (string, int64,
 		from = time.Now().Unix()
 	}
 
-	return userID, from, limit
+	if err != nil {
+		// If it returns error, it will assume the default behavior(e.g. isBlocked=false).
+		fmt.Println("ERROR:", err)
+		return userID, from, limit, false, err
```

Comment:
Thinking that we already handle the error here, we could not leak it to the callers, because there is no need to handle it.

Reply:
Sure!